modules/client/examples.md

```diff
@@ -1753,6 +1753,149 @@ Returns:
 */
 ```
 
+### Prepare the installation of a token voting plugin
+
+```ts
+import {
+  ContextPlugin,
+  PrepareInstallationStep,
+  TokenVotingClient,
+  TokenVotingPluginPrepareInstallationParams,
+  VotingMode,
+} from "@aragon/sdk-client";
+import { context } from "../00-setup/00-getting-started";
```

Comment:
should we really start importing in the examples from other examples? I assume seeing an example that all the necessary code is present

Reply:
Its just the declaration of the context, initially I did the examples withoud importing the context but it lead to super big examples in which the first pard didnt add anything useful